string.md

```diff
@@ -37,35 +37,12 @@ When inserting a string:
 
 ## Formats
 
-A `STRING` column accepts Unicode string literals, hexadecimal string literals, and escape strings. 
+A `STRING` constant can be expressed using a variety of formats,
```

Comment:
Let's retitle this section `Syntax` and, for consistency, rephrase the content a bit:

```
Values of type `STRING` can be expressed in a variety of formats. See [string literals](sql-constants.html#string-literals) for more details.
```